doc/man/dogecoin-cli.1

```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
```

Comment:
missing cli documentation for the `listunspent` enhancements?

Reply:
Ah probably my binaries are out of date

Reply:
Nope, the cli docs doesn't cover RPC commands it turns out